documentation/release_3.1.htm

```diff
@@ -372,6 +372,7 @@ <h3>Other code changes</h3>
   All derived classes of <code>CListEvent</code> will need to implement it.</li>
 <li>We removed <code>SeparableMetzArrayFilter::discrete_fourier_transform()</code>. Now <code>stir::fourier</code> is used. No need to comply with the Licence of "Numerical Recipies" anymore.</li>
 </li>
+<li>Added a tests Make target so that `make tests` will compile all tests.</li>
```

Comment:
```suggestion
<li>Added a <tt>tests</tt> build target so that <tt>make tests</tt> (or whatever is appropriate for your build system) will build all tests (not the other executables or Python etc libraries) .</li>
```
currently still using html.